### DISCUSSION TITLE
Trouble stitching mp4 files together taken on same phone, on same day.

### DISCUSSION THREAD

**Discussion by kklasmeier** [mifi/lossless-cut#1400] [Support]
On my phone, I took family videos. Trying to merge them together into a single MP4 file and finding problems in the app.

Check compatibility complains each file is encoded differently than one and other. 

Track 0 mismatch: r_frame_rate 30/1 != 30000/1001
Track 0 mismatch: width 800 != 1920
etc.

Furthermore, I cannot determine how many different combinations I have, so I can't even try to group them.

When I view the files individually, they work and look great and they all look 100% the same as each other. Any ideas how I can stitch these together? I have 12,000 files that I want to stitch and I am at a loss given these compatibility alerts.

**Comment by mifi** [comment]
You could write some script to use ffmpeg to generate a report of all the files and their codec parameters, and then group them based on that. This isn't something that LosslessCut currently supports. Maybe in the future but you're the first persron who requested this so far

**Comment by eleazar7** [comment]
I think i have the same problem with Losslesscut. 

i tried to stitch together 5 videos, which were all exported with the same settings and preset, but Losslesscut says they are different. 

solution: 
I use Avidemux, and it works fine. Sometimes I have to merge first in MKV format and then re-import and mux in MP4 for the final output. 

Having this in Losslesscut would really be a big deal.

**Reply by mifi** [reply]
Are you getting an error duing merge or just warnings?

**Reply by eleazar7** [reply]
Well, at first i get warning message, but when i merge the videos together, the audio and video are messed up. 
when use Avidemux app with same video files, it works just fine. 

sometimes with Avidemux, I have to mux the videos into MKV format first, and then rewrap in mp4 for it to work but not all the time. sometimes i can just go right to mp4 and works great.

**Reply by mifi** [reply]
Probably a bug or limitation in the way that LosslessCut/FFmpeg concats files then, related to your particular files. I don't know how to solve it unfortunately.
